src/com/goide/actions/fmt/GoImportsFileAction.java

```diff
@@ -64,7 +62,6 @@ public boolean doSomething(@NotNull PsiFile file,
     String filePath = virtualFile.getCanonicalPath();
     assert filePath != null;
 
-    GeneralCommandLine commandLine = new GeneralCommandLine();
     String sdkHome = GoSdkService.getInstance(project).getSdkHomePath(ModuleUtilCore.findModuleForPsiElement(file));
     if (StringUtil.isEmpty(sdkHome)) {
```

Comment:
Btw, I think we can extract checking sdk in GoExternalToolsAction. Just check in update method that sdkPath is not empty and make setEnabled(false) if it's empty.